src/index.test.js

```diff
@@ -95,8 +97,12 @@ test('always shows dropdown with inline search Input', t => {
 test('keeps dropdown open for showDropdown: always', t => {
   const { tree } = t.context
   const wrapper = mount(<DropdownTreeSelect id={dropdownId} data={tree} showDropdown="always" />)
-  wrapper.instance().handleClick()
-  t.true(wrapper.state().showDropdown)
+  wrapper.find('.search').simulate('click')
+  t.true(wrapper.exists('.dropdown-content'))
+  const event = new MouseEvent('click', { bubbles: true, cancelable: true })
+  global.document.dispatchEvent(event)
```

Comment:
I kept the `simulate('click')` because the old test had it, but it could be removed. The purpose of the dispatch is to ensure that, if the user clicks outside the dropdown, it stays open.